<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>miniWinamp</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="4">
       <widget class="QPushButton" name="buttonPlay">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons/play-button.png</normaloff>icons/play-button.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QPushButton" name="buttonPause">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons/stop.png</normaloff>icons/stop.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSlider" name="volSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="buttonNext">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons/next.png</normaloff>icons/next.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="buttonBack">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons/back.png</normaloff>icons/back.png</iconset>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <spacer name="spaceBetweenButtons">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>120</width>
          <height>1</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="iconVolume">
        <property name="text">
         <string>icon</string>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="currentTimeLabel">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>0:00</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSlider" name="horizontalSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="fullTimeLabel">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>0:00</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit">
      <property name="accessibleDescription">
       <string/>
      </property>
      <property name="text">
       <string>Поиск трека</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListView" name="listTracks"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>335</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufdsfsd">
    <property name="title">
     <string>Стили</string>
    </property>
    <widget class="QMenu" name="menuStyles">
     <property name="title">
      <string>Styles/Стили</string>
     </property>
     <addaction name="StyleDefault"/>
     <addaction name="StyleBlack"/>
     <addaction name="StyleOrange"/>
    </widget>
    <addaction name="menuStyles"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action_3"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menufdsfsd"/>
  </widget>
  <action name="StyleBlack">
   <property name="text">
    <string>Тёмный</string>
   </property>
  </action>
  <action name="StyleDefault">
   <property name="text">
    <string>Светлый (по умолчанию)</string>
   </property>
  </action>
  <action name="StyleOrange">
   <property name="text">
    <string>Оранжевый</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
